datasets/my/investor_alert_list/crawler.py

```diff
@@ -0,0 +1,56 @@
+import json
+from zavod import Context, helpers as h
+
+
+def crawl_item(input_dict: dict, context: Context):
+
+    name = input_dict.pop("name")
+
+    # If it's a potential clone, we remove the "potential clone" from the name
+    name = name.replace("Potential clone entity – ", "")
+
+    entity = context.make("LegalEntity")
+    entity.id = context.make_id(name)
+    entity.add("topics", "sanction")
```

Comment:
these aren't true sanctions in the sense we mean and our customers look for. I've updated the description of what we mean by sanctions at https://www.opensanctions.org/datasets/sanctions/

Let's make these `crime.fin` on the basis of https://www.sc.com.my/resources/media/media-release/better-investor-protection-with-amended-securities-laws-frequently-asked-questions-on-amendments-to-securities-laws